standards/application/relay-reliability.md

```diff
@@ -0,0 +1,192 @@
+---
+title: TRANSPORT-RELIABILITY
+name: Waku Transport Reliability 
+category: Standards Track
+tags: [reliability, application]
+editor: Kaichao Sun <kaichao@status.im>
+contributors:
+  - Richard Ramos <richard@status.im>
+---
+
+## Abstract
+
+Waku is efficient for routing messages with its transport layer like [Relay protocol](https://github.com/vacp2p/rfc-index/blob/main/waku/standards/core/11/relay.md) and [Lightpush](https://github.com/vacp2p/rfc-index/blob/main/waku/standards/core/19/lightpush.md) / [Filter](https://github.com/vacp2p/rfc-index/blob/main/waku/standards/core/12/filter.md) protocol. But there's no guarantee that a message going through Waku will reach to its destination. For example, the receiver in a chat application may miss messages when network issue happens at either sender or receiver side. 
+
+In general, a message in Waku network includes 3 status from sender's perspective:
+
+- **outgoing**, the message is posted by its creator but no confirmations from other nodes yet
```

Comment:
I would define either sender/publisher (or both) and try to stick to the same terminology throughout. "creator" is a new term here and I think can be consistent in using "publisher" here.